setup.py

```diff
@@ -6,6 +6,7 @@
     'aiohttp<4',
     'dataclasses',
     'requests<3',
+    'pandas',
```

Comment:
pandas would be quite useful to use here to keep storing data for example if someone wanted to keep streaming data and save the data for later use for backtesting for example.

Reply:
I am very wary of prescribing use-cases with this API. The purpose with this repo is to just provide the tool, not recommend an application for it :)

Reply:
Got it, I think this was in order to include an example on how to do that but this can be removed and stick with the lower level list/dict. I will remove this.